### PR TITLE
made tokenization_roformer independent of bert

### PR DESCRIPTION
**Pull request by naveennamani**
# What does this PR do?
Relates to issue #19303 

Fixes #19303 

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@patrickvonplaten

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Gently pinging @sgugger @ArthurZucker for re-distribution here

**Comment by ArthurZucker** [comment]
Will take care of it 👍

**Comment by sgugger** [comment]
Don't worrk @ArthurZucker I'll review :-) This is linked to #19303 and I should have been tagged, not Patrick :-)

**Comment by naveennamani** [comment]
@sgugger when I opened the PR, in the PR template it showed me @patrickvonplaten is looking after roformer which I modified, so tagged him for the review.

**Comment by sgugger** [comment]
No worries at all @naveennamani , but as @patrickvonplaten is very busy on plenty of other things and you PR should not make any actual changes so Patrck's input won't be necessary. I'll review it tomorrow :-)

File: src/transformers/models/roformer/tokenization_roformer.py [hunk 1]
```diff
@@ -66,6 +66,236 @@
 }
 
 
+# copied from ..bert.tokenization_bert
```
Comment (sgugger) [inline src/transformers/models/roformer/tokenization_roformer.py:69]:
```suggestion
# Copied from transformers.models.bert.tokenization_bert.load_vocab
```
The copied from statement needs to begin with a capital and use the full absolute path. Showed here above, it's the same for all other copied from in this file.

**Review by sgugger — COMMENTED** [review]
Thanks a lot for your work on this! There is just a problem of syntax in the copied from statements to fix, but otherwise should be good to go!

**Review by sgugger — COMMENTED** [review]
As said before, the copied from statement needs a capital C: `# Copied from`. Could you update this as well?

**Comment by naveennamani** [comment]
I missed it somehow, fixed it

**Review by sgugger — APPROVED** [review]
Perfect, thanks a lot!

**Pull request merged**